packages/logging-controller/CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
```

Comment:
I think we will want to start this off with an empty changelog for now. It's customary for us to update the changelog when we issue a new release rather than have it already ready to go prior to the new release.